src/styles/base/_theme-css-vars.scss

```
@@ -0,0 +1,74 @@
+// Themes
+:root,
+.mc-theme-dark {
+  // Main colors
+  --mc-theme-background:              #{$mc-color-dark};
+  --mc-theme-text:                    #{$mc-color-light};
+  --mc-theme-separator:               #{rgba($mc-color-light, 0.3)};
+  --mc-theme-border-contrast:         #{rgba($mc-color-light, 0.5)};
+  --mc-theme-border-contrast-hover:   #{$mc-color-light};
+  // Dropdown colors
+  --mc-theme-dropdown-backdrop:       #{rgba($mc-color-dark, 0.9)};
+  --mc-theme-dropdown-border:         #{$mc-color-gray-200};
+  --mc-theme-dropdown-cont-bg:        #{$mc-color-gray-100};
+  --mc-theme-dropdown-cont-shadow:    #{$mc-color-dark};
+  --mc-theme-dropdown-itm-hover-bg:   #{$mc-color-gray-200};
+  // Tooltip colors
+  --mc-theme-tooltip-cont-bg:       #{$mc-color-gray-200};
+  --mc-theme-tooltip-cont-border:   #{$mc-color-gray-400};
+  --mc-theme-tooltip-arrow:         #{$mc-color-gray-400};
+  --mc-theme-tooltip-arrow-after:   #{$mc-color-gray-200};
+  // Checkbox colors
+  --mc-theme-checkbox-fb-bg:              #{$mc-color-gray-100};
+  --mc-theme-checkbox-fb-border:          #{$mc-color-gray-200};
+  --mc-theme-checkbox-fb-after-bg:        #{$mc-color-light};
```

same with `-after`, this is actually defining the color of the check. 
```suggestion
  --mc-theme-checkbox-check:        #{$mc-color-light};
```